libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -567,30 +469,26 @@ private void addBottomSheetListener() {
     }
   }
 
-  private void initializeClickListeners() {
-    alertView.setOnClickListener(new View.OnClickListener() {
-      @Override
-      public void onClick(View view) {
-        if (((AlertView) view).getAlertText().equals(NavigationConstants.REPORT_PROBLEM)) {
-          navigationViewModel.recordFeedback(FeedbackEvent.FEEDBACK_SOURCE_REROUTE);
-          showFeedbackBottomSheet();
-        }
-        alertView.hide();
-      }
-    });
-    soundFab.setVisibility(VISIBLE);
-    soundFab.setOnClickListener(new View.OnClickListener() {
+  private void initializeButtons() {
+    setupSoundButton();
+    setupFeedbackButton();
+  }
+
+  private void setupFeedbackButton() {
+    feedbackButton.addOnClickListener(new OnClickListener() {
       @Override
       public void onClick(View view) {
-        navigationViewModel.setMuted(toggleMute());
+        navigationViewModel.recordFeedback(FeedbackEvent.FEEDBACK_SOURCE_UI);
+        showFeedbackBottomSheet();
       }
     });
-    feedbackFab.setVisibility(VISIBLE);
-    feedbackFab.setOnClickListener(new View.OnClickListener() {
+  }
+
+  private void setupSoundButton() {
+    soundButton.addOnClickListener(new View.OnClickListener() {
```

Comment:
Same here ☝️

Reply:
Minor thing - `View.` is not needed (`OnClickListener` is enough as in `setupFeedbackButton` ☝️)